### ISSUE TITLE
Can't reorder mods in Skyrim SE or Fallout 4

### ISSUE THREAD

**Issue by stargatedalek** [Nexus-Mods/Nexus-Mod-Manager#318]
Mysteriously started give or take three to five hours ago.

I have the latest version and have been using it for around a week now.

On launching I get an error:

The following updates failed:
Nexus Mod Manager Updater: Could not get version information from the update server

This in itself is not a huge concern, there are no newer updates, and if there were I could install them manually. I am still able to log into my account etc. and download mods normally.

When this error started however it came with the inability to reorder my plug ins. The vertical green arrows are greyed out, as are the export/import load order buttons.

**Comment by stargatedalek** [comment]
I will note that LOOT can still reorder plug-ins, but I have no control over it, and I don't trust LOOT to get it right every time.

**Comment by 27Bslash6** [comment]
Same issue

**Comment by alexrsosa** [comment]
same issue

**Comment by squid-box** [comment]
I always get suspicious when people won't specify what they think "the latest version" is. The load order issue was fixed and works in the following versions:
* 0.65.3
* 0.65.4
* 0.65.5
* 0.65.6
* 0.65.7

Then I broke it in 0.65.8, and it was fixed in 0.65.9 again.

So if you're not using one of these versions, just grab [NMM 0.65.9](https://github.com/Nexus-Mods/Nexus-Mod-Manager/releases/tag/0.65.9) and it should work fine.
